lib/surface/components/form/submit.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Surface.Components.Form.Submit do
+  @moduledoc """
+  Generates a submit button to send the form.
+
+  All options are forwarded to the underlying `Phoenix.HTML.Form.submit/3`
+  """
+
+  use Surface.Component
+
+  import Phoenix.HTML.Form, only: [submit: 2]
+
+  @doc "The text to be used in the button"
+  property text, :string
```

Comment:
We should either call it `label` to keep it consistent with the other components or change all `label`s to `text`. Although the helpers call it `text`, I believe `label` describes better. WDYT?

Reply:
Makes sense. Phoenix calls it value which I don't think makes sense here, so I was debating what to call it.